rnb-cli/src/tools/react-native.ts

```diff
@@ -98,11 +98,11 @@ export async function renameReactNativeApp(
     `android/app/_BUCK`,
     `android/app/BUCK`,
     `android/app/build.gradle`,
-    `android/app/src/debug/java/${newPath}/ReactNativeFlipper.java`,
+    // `android/app/src/debug/java/${newPath}/ReactNativeFlipper.java`,
```

Comment:
Do you think this comment is intended?

Reply:
Great! I deleted the commented line, that file is not necessary anymore on RN 0.73: https://react-native-community.github.io/upgrade-helper/?from=0.72.5&to=0.73.5